src/renderer/shared/api/xcm/xcmService.ts

```diff
@@ -143,94 +181,115 @@ const getRelativeAssetLocation = (
   api: ApiPromise,
   amount: BN,
   assetLocation?: LocalMultiLocation,
-): VersionedMultiAsset | undefined => {
+): Object | undefined => {
   if (!assetLocation) return;
 
   const { parachainId: _, ...location } = assetLocation;
 
-  return api.createType('VersionedMultiAsset', {
-    V2: {
-      id: {
-        Concrete: {
-          parents: 0,
-          interior: Object.values(location).length ? createJunctionFromObject(location) : 'Here',
+  return {
+    V2: [
+      {
+        id: {
+          Concrete: {
+            parents: 0,
+            interior: createJunctionFromObject(location),
+          },
+        },
+        fun: {
+          Fungible: amount.toNumber(),
```

Comment:
Number can only safely store up to 53 bits.
https://app.clickup.com/t/862kd1bj9
need BN